packages/main/src/themes/FileUploader.css

```diff
@@ -0,0 +1,44 @@
+:host {
+	display: inline-block;
+}
+
+.ui5-file-uploader-root {
+	position: relative;
+}
+
+.ui5-file-uploader-root input[type=file] {
+	opacity: 0;
+	position: absolute;
+	top: 0;
+	left: 0;
+	height: 100%;
+	width: 100%;
+}
+
+.ui5-file-uploader-root input[type=file]:not([disabled]) {
+	cursor: pointer;
+}
+
+.ui5-file-uploader-mask {
+	display: flex;
+}
+
+.ui5-file-uploader-mask ui5-input {
+	margin-right: 0.25rem;
+}
+
+:host(:not([value-state]):not([disabled]):hover) ui5-input {
+	background-color: var(--sapField_Hover_Background);
+	border: 1px solid var(--sapField_Hover_BorderColor);
+}
+
+:host(:not([disabled]):hover) ui5-button {
```

Comment:
This is very interesting. It was intended to make the button have a hover effect. The problem is that the user actually hovers, clicks, etc... on the transparent file input and the button does not react. Please try to rework this (and the next) selector to target slotted buttons, if possible.